ch-covidcertificate-backend-verifier/ch-covidcertificate-backend-verifier-ws/src/main/java/ch/admin/bag/covidcertificate/backend/verifier/ws/config/WSSchedulingConfig.java

```diff
@@ -23,6 +23,9 @@
     public WSSchedulingConfig(ApiKeyConfig apiKeyConfig, AppTokenDataService appTokenDataService) {
         this.apiKeyConfig = apiKeyConfig;
         this.appTokenDataService = appTokenDataService;
+
+        // Load tokens at start up
+        updateAppTokens();
```

Comment:
would probably better using @PostConstruct https://www.baeldung.com/spring-postconstruct-predestroy 
But should work aswell

Reply:
Adjusted it